judge/migrations/0055_add_performance_points.py

```diff
@@ -11,13 +11,15 @@
 def gen_pp(apps, schema_editor):
     Profile = apps.get_model('judge', 'Profile')
     Problem = apps.get_model('judge', 'Problem')
-    table = (lambda x: [pow(x, i) for i in xrange(100)])(getattr(settings, 'PP_STEP', 0.95))
-    bonus_function =  getattr(settings, 'PP_BONUS_FUNCTION', lambda n: 300 * (1 - 0.997 ** n))
+    _pp_step = getattr(settings, 'DMOJ_PP_STEP', 0.95)
+    table = (lambda x: [pow(x, i) for i in xrange(getattr(settings, 'DMOJ_PP_ENTRIES', 100))])(_pp_step)
```

Comment:
Why do we still need this `lambda`? It only existed in the old code because whoever wrote it does not want to have a variable.